README.md

```diff
@@ -7,6 +7,8 @@ https://dev.azure.com/ansible/community.postgres/_apis/build/status/CI?branchNam
 
 We follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our interactions within this project.
 
+If you encounter abusive behavior violating the [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html), please refer to the [policy violations](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#policy-violations) section of the Code of Conduct for information on how to raise a complaint.
+
 ## Contributing to this collection
 
 The content of this collection are made by [people](CONTRIBUTORS) like you.
```

Comment:
```suggestion
The contents of this collection are made by [people](CONTRIBUTORS) like you.
```

Reply:
It doesn't allow to merge it here, so I'll create a new PR, thanks!